### DISCUSSION TITLE
How to change python3 path

### DISCUSSION THREAD

**Discussion by vijayank88** [chipsalliance/Surelog#3400] [General]
Hi,

I am trying to install surelog and fail with following error: (Ubuntu 20.04 OS)
```
-- Found Python3: /usr/local/bin/python3.5 (found version "3.5.0") found components: Interpreter 
-- Python3_EXECUTABLE = /usr/local/bin/python3.5
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named 'orderedmultidict'
CMake Error at third_party/UHDM/CMakeLists.txt:36 (message):
  The "orderedmultidict" Python3 package is not installed.  To install use
  following command: "pip3 install orderedmultidict".
```
My default version of python3 is pointing 3.8.10. But somehow the `make` command pointing to `/usr/local/bin/python3.5`. How to fix this issue?
Required packages already installed.

**Comment by hs-apotell** [comment]
This is not a Surelog issue. This is specific to how you have configured your machine. cmake/make will use whatever the _which_ command reports.

**Reply by vijayank88** [reply]
I resolved the issue by using `/usr/bin/python3.7` executable path, instead of script default search. Thanks.
